.circleci/config.yml

```diff
@@ -0,0 +1,26 @@
+version: 2
+jobs:
+  deploy:
+    docker:
+      - image: circleci/python:3-node
+    steps:
+      - checkout
+      - run: yarn install --ignore-engines
+
+      # TODO: Maybe fix this? Or at least delete this comment
+      - run: CI=false yarn run build
```

Comment:
Without `CI=false`, all warnings become errors and it won't build. Not sure how ya'll front end folks wanna handle this.